packages/maskbook/src/plugins/Furucombo/UI/FurucomboView.tsx

```diff
@@ -0,0 +1,165 @@
+import { makeStyles } from '@masknet/theme'
+import { useChainId } from '@masknet/web3-shared'
+import { Card, CardContent, CardActions, Tabs, Tab, Typography, Link, Paper } from '@material-ui/core'
+import { useState } from 'react'
+import { FurucomboIcon } from '../../../resources/FurucomboIcon'
+import { MaskTextIcon } from '../../../resources/MaskIcon'
+import { useI18N } from '../../../utils/i18n-next-ui'
+import { useFetchPools } from '../hooks/usePool'
+import type { Investable } from '../types'
+import { InvestmentsView } from './InvestmentsView'
+import { Poolview } from './PoolView'
+
+const useStyles = makeStyles()((theme) => ({
+    root: {
+        fontFamily: 'Muli,Helvetica,Arial,"sans-serif"',
+        width: '100%',
+        boxShadow: 'none',
+        border: `solid 1px ${theme.palette.divider}`,
+        padding: 0,
+    },
+    content: {
+        width: '100%',
+        height: 'var(--contentHeight)',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'space-around',
+        padding: '0 !important',
+    },
+    body: {
+        // flex: 1,
+        // overflow: 'auto',
+        // maxHeight: 350,
+        // borderRadius: 0,
+        // scrollbarWidth: 'none',
+        // '&::-webkit-scrollbar': {
+        //     display: 'none',
+        // },
+    },
+    tabs: {
+        borderTop: `solid 1px ${theme.palette.divider}`,
+        borderBottom: `solid 1px ${theme.palette.divider}`,
+        backgroundColor: `#1b1b21`,
+    },
+    tab: {
+        fontFamily: 'inherit',
+        color: 'white',
+    },
+    footer: {
+        marginTop: -1, // merge duplicate borders
+        zIndex: 1,
+        position: 'relative',
+        borderTop: `solid 1px ${theme.palette.divider}`,
+        justifyContent: 'space-between',
+    },
+    footnote: {
+        fontSize: 10,
+        marginRight: theme.spacing(1),
+    },
+    footLink: {
+        cursor: 'pointer',
+        marginRight: theme.spacing(0.5),
+        '&:last-child': {
+            marginRight: 0,
+        },
+    },
+    footMenu: {
+        color: theme.palette.text.secondary,
+        fontSize: 10,
+        display: 'flex',
+        alignItems: 'center',
+    },
+    footName: {
+        marginLeft: theme.spacing(0.5),
+    },
+    maskbook: {
+        width: 40,
+        height: 10,
+    },
+    furucombo: {
+        height: 16,
+        margin: theme.spacing(-0.5, 0.25),
+    },
+}))
+
+interface PoolViewProps {
+    category: string
+    chainId: string
```

Comment:
Why do the `chainId` and `category` are not used in the component? It only renders from fix data?

Reply:
Indeed, they have no reason to be.